### ISSUE TITLE
[Feature request] Add some basic stock portfolio management capabilities

### ISSUE THREAD

**Issue by GLLM** [firefly-iii/firefly-iii#844]
Hi !

I've been looking at your project and I find it very good, nice, polished/well designed !

One thing is missing (according to me) in it : I have small positions on some compagnies' shares ... and I cannot monitor them in Firefly III.

What I would need would be the ability to :
- add a position for a given share : number of shares, compagnies, transaction date, price, portfolio name
- remove / unwind a position at a given date/price for a given amount in a given portfolio
- monitor the portfolio value per day / graph the value / P&L evolution
- optional but reaaaaaaally nice : take the stock prices either on google or yahoo finance

This way, I'd be able to monitor my my budget/income/expenses/assets (casha ccount, deposit with interests, stock positions) ... basically : all I own/owe :)

Just a wild wish ! But we never know :)

Thanks and keep up the very good work !

GLLM

**Comment by JC5** [comment]
Thanks, I appreciate the compliments!

I've thought about it before and I've decided not to extend Firefly with this. It's quite a stretch from the original goal of Firefly which is really focussed on household finances.

I have some shares myself and I must admit I've toyed with the idea of extending FF to be able to track them, but it doesn't really "fit". 

At the moment, my focus for FF is to keep it as feature-poor as possible (strange as this may sound) and on making sure the existing features actually work.

It's not something I don't want to build, but it's something I wouldn't include in Firefly.
